Examples/Io/Root/src/RootAthenaDumpReader.cpp

```diff
@@ -14,10 +14,60 @@
 #include "ActsExamples/EventData/Cluster.hpp"
 #include "ActsExamples/EventData/GeometryContainers.hpp"
 #include "ActsExamples/EventData/IndexSourceLink.hpp"
+#include <ActsExamples/Digitization/MeasurementCreation.hpp>
+
+#include <cmath>
 
 #include <TChain.h>
 #include <boost/container/static_vector.hpp>
 
+class BarcodeConstructor {
+  /// Particles with barcodes larger then this value are considered to be
+  /// secondary particles
+  constexpr static int s_maxBarcodeForPrimary = 200000;
```

Comment:
does this magic number have an origin or is it an arbitrary high number?

Reply:
Hmm I can link to Athena code where it comes from...